rules/azure_signin_rules/azure_failed_signins.yml

```diff
@@ -0,0 +1,160 @@
+AnalysisType: rule
+Filename: azure_failed_signins.py
+RuleID: "Azure.SignIn.ManyFailedSignIns"
+DisplayName: "Azure Many Failed SignIns"
+Enabled: true
+# Ten Failed Sign-Ins(Threshold) in Ten Minutes(DedupPeriodMinutes)
+Threshold: 10
+DedupPeriodMinutes: 10
+LogTypes:
+  - Azure.SignIn
+Severity: Low
```

Comment:
if we set the threshold to ten do we still want the sev to be low?

Reply:
yeah, that makes sense to me. Severity updated in latest commit